zipline/assets/roll_finder.py

```diff
@@ -226,3 +229,8 @@ def _active_contract(self, oc, front, back, dt):
             if back_vol > front_vol:
                 return back
         return front
+
+    def get_contract_center(self, root_symbol, dt, offset):
+        day = self.trading_calendar.day
+        rolls = self.get_rolls(root_symbol, dt, dt + (180 * day), offset)
+        return self.asset_finder.retrieve_asset(rolls[0][0])
```

Comment:
The number `180` is up for discussion as what it should be, or what the bounds on the `get_rolls` call should be. Either way this should probably be defined as a constant somewhere.